.github/workflows/snyk_push.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Snyk Clojure Merge
+
+on: push
```

Comment:
Why does this need a separate action rather than configuring multiple triggers?
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-activity-types-and-filters-with-multiple-events

Reply:
I kept them separate because the `push` variety doesn't need the label check, since you can only push branches to the repo if you have write access, so the label isn't necessary. This follows the templates that other teams have been using.

Reply:
^ that doc suggests you could do
```
on:
  push:
  pull_request_target:
    types: [opened, reopened, synchronize, labeled]
```
and the types would only apply to the PR trigger

Reply:
Right, I'm talking about this conditional: https://github.com/puppetlabs/puppetserver/blob/70250ea6e25157b63894415cbf050506427dcede/.github/workflows/snyk.yml#L11. I think I saw another example somewhere (maybe the vanagon examples?) that had both in one file, but still needed two separate workflows because we want to gate it running more specifically than just on the triggers, in a way that only applies to PRs. Maybe I'm still missing something? But I almost think this is clearer regardless 🤷‍♀️

Reply:
ahh, okay. ¯\_(ツ)_/¯